### ISSUE TITLE
in Gridcell align='middle' not working

### ISSUE THREAD

**Issue by nilacseruet** [jamesmfriedman/rmwc#266]
I am trying to put my gridcell content in middle. I tested with mdc, its working fine. But on my react, I am using rmwc and trying to do align middle, but it's not working.

Here is my code:
```
<Grid>
      <GridCell span="12" align='middle'>Test content</GridCell>
</Grid>
```
I saw from Inspect element, its shows the div class like belows
```
<div class=" mdc-layout-grid__cell--align-middle">

```
How can I make it html tag(```<div class="" align='middle'>```), not into the classname.
 
I am using react 15.6. 
rmcw: 1.7.5

**Comment by jamesmfriedman** [comment]
Unfortunately, this is a little difficult at the moment.

It actually does work, but requires some styling. If you look at the material docs they actually have custom sizing on the interior grid element. Use some CSS to set the height of the inner grid. I'll also go ahead and add an enhancement to The grid to allow you to return your own GridInner component with custom styles.

https://codesandbox.io/s/31jkvy6k35

**Comment by jamesmfriedman** [comment]
Added a fix so you can specify your own GridInner and give it a height. Will be released with 1.8

**Comment by nilacseruet** [comment]
Great, thanks a lot.
